WinRT.Runtime/Interop/IAgileReference.net5.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace WinRT.Interop
+{
+    [WindowsRuntimeType]
+    [Guid("C03F6A43-65A4-9818-987E-E0B810D2A6F2")]
+    internal interface IAgileReference
+    {
+        IObjectReference Resolve(Guid riid);
+    }
+
+    [WindowsRuntimeType]
+    [Guid("94ea2b94-e9cc-49e0-c0ff-ee64ca8f5b90")]
+    public interface IAgileObject
+    {
+        public static readonly Guid IID = Guid.Parse("94ea2b94-e9cc-49e0-c0ff-ee64ca8f5b90");
+    }
+
+    [WindowsRuntimeType]
+    [Guid("00000146-0000-0000-C000-000000000046")]
+    internal interface IGlobalInterfaceTable
+    {
+        IntPtr RegisterInterfaceInGlobal(IObjectReference objRef, Guid riid);
+        void RevokeInterfaceFromGlobal(IntPtr cookie);
+        IObjectReference GetInterfaceFromGlobal(IntPtr cookie, Guid riid);
+    }
+}
+
+namespace ABI.WinRT.Interop
+{
+    using global::WinRT;
+    using WinRT.Interop;
+
+    [DynamicInterfaceCastableImplementation]
+    [Guid("C03F6A43-65A4-9818-987E-E0B810D2A6F2")]
+    internal unsafe interface IAgileReference : global::WinRT.Interop.IAgileReference
+    {
+        [Guid("C03F6A43-65A4-9818-987E-E0B810D2A6F2")]
+        public struct Vftbl
+        {
+            public global::WinRT.Interop.IUnknownVftbl IUnknownVftbl;
+            private void* _Resolve;
+            public delegate* stdcall<IntPtr, ref Guid, out IntPtr, int> Resolve { get => (delegate* stdcall<IntPtr, ref Guid, out IntPtr, int>)_Resolve; set => _Resolve = value; }
+
+            public static readonly Vftbl AbiToProjectionVftable;
+            public static readonly IntPtr AbiToProjectionVftablePtr;
+
+#if NETSTANDARD2_0
```

Comment:
Do we need netstandard compat in a net5 file?

Reply:
How much diff is there to merge both files?

Reply:
I think I fixed it in a newer commit.